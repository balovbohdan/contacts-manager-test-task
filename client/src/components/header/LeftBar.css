.main {
    width: 50%;
}

.logo {
    width: 40px;
    height:40px;

    cursor: pointer;

    background-size: contain;
    background-repeat: no-repeat;
    background-position: center center;
    background-image: url(~@img/logo.svg);
}