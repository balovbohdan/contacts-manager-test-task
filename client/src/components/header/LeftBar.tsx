import * as React from 'react';
import {Link} from 'react-router-dom';

import {paths} from '@routes';

import css from './LeftBar.css';

export const LeftBar = () =>
    <div className={css.main}>
        <Logo/>
    </div>;

const Logo = () =>
    <Link to={paths.home}>
        <div className={css.logo}/>
    </Link>;