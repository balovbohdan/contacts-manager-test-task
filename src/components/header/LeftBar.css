.main {
    width: 50%;
}

.logo {
    color: var(--txt-color-yellow);
    font-family: var(--font-marker);
}