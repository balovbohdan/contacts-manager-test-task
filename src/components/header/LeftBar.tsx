import * as React from 'react';

import css from './LeftBar.css';

export const LeftBar = () =>
    <div className={css.main}>
        <Logo/>
    </div>;

const Logo = () =>
    <h3 className={css.logo}>CM</h3>;