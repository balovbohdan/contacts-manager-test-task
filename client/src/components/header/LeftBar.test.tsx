import {shallow} from 'enzyme';
import * as React from 'react';

import {LeftBar} from './LeftBar';

describe('LeftBar of Header', () => {
    it('Should match snapshot', () => {
        const bar = shallow(<LeftBar/>);

        expect(bar).toMatchSnapshot();
    });
});