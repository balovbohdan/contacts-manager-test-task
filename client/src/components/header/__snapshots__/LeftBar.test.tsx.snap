// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeftBar of Header Should match snapshot 1`] = `
<div
  className="main"
>
  <Logo />
</div>
`;
